### PR TITLE
Add templates migration

### PR DESCRIPTION
**Pull request by iskhakov**
# What this PR does

## Checklist

- [ ] Unit, integration, and e2e (if applicable) tests updated
- [ ] Documentation added (or `pr:no public docs` PR label added if not required)
- [ ] `CHANGELOG.md` updated (or `pr:no changelog` PR label added if not required)

### CODE REVIEW

File: engine/apps/alerts/migrations/0015_auto_20230428_1148.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                    # Value is not set, set it to default in backend_templates
+                    value = alert_receive_channel.get_default_template_attribute(
+                        backend_id, field
+                    )
+                    backend_templates[field] = value
+            messaging_backends_templates[backend_id] = backend_templates
+        print(
+            f"{alert_receive_channel}: Setting messaging_backend_templates: \n {messaging_backends_templates}"
+        )
+        # TODO: uncomment to save
```
Comment (Konstantinov-Innokentii) [inline engine/apps/alerts/migrations/0015_auto_20230428_1148.py:75]: Could we use batch_update here?

**Comment by Konstantinov-Innokentii** [comment]
One thing which came to my mind - if alert receive channel is created via terraform, terraform will show diff after this migration will executed and tries to return null values.

File: engine/apps/alerts/migrations/0015_auto_20230428_1148.py (hunk 1 above)
Comment (iskhakov) [reply engine/apps/alerts/migrations/0015_auto_20230428_1148.py:75]: we have only 500 entries

File: engine/apps/alerts/migrations/0015_auto_20230428_1148.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("alerts", "0014_alertreceivechannel_restricted_at"),
+    ]
+
+    operations = [
+        # TODO: uncomment
+        # migrations.RunPython(),
```
[the remark this answers is not in the document]
Comment (iskhakov) [reply engine/apps/alerts/migrations/0015_auto_20230428_1148.py:87]: It won't be revertable:(

**Comment by iskhakov** [comment]
not relevant, it causes tf diff for customers

**Pull request closed without merging**